datajunction-server/alembic/versions/2025_01_24_0020-bec3296d7537_add_custom_metadata_field_to_nodes.py

```diff
@@ -0,0 +1,28 @@
+"""Add custom metadata field to nodes.
```

Comment:
Haha, we're gonna need a lot of rebasing depending on who merges their PR first ;)

Reply:
You can go first. I forgot to add this to the YAML project gears.